README.md

```diff
@@ -1,22 +1,63 @@
-# Ionic Framework
-
-[Ionic Framework](https://ionicframework.com/) is the open-source mobile app development framework that makes it easy to
-build top quality native and progressive web apps with web technologies.
-
-Ionic Framework is based on [Web Components](https://www.webcomponents.org/introduction) and comes with many significant performance, usability, and feature improvements over the past versions.
-
-## Looking for the Ionic Framework v6 beta?
-
-[Click here to get started!](https://github.com/ionic-team/ionic-framework/blob/next/BETA.md)
+<p align="center">
+  <a href="#">
+    <img alt="Ionic" src="https://ionic.io/_next/image?url=https%3A%2F%2Fimages.prismic.io%2Fionicframeworkcom%2F66cfdbef-e59d-463a-8e24-12cb233e9d97_ionic%2Blogo%2Bblue.png&w=128&q=75" width="60" />
```

Comment:
Can we commit this logo to the repo instead of loading it from a separate website?

Reply:
Sure thing 👍  I was unsure if we wanted the assets to be managed by the web team, but easy to remedy if they need changes in the future & hosting within Github is likely more reliable.